flake.nix

```
@@ -5,8 +5,9 @@
   inputs.nixpkgs-regression.url = "github:NixOS/nixpkgs/215d4d0fd80ca5163643b03a33fde804a29cc1e2";
   inputs.lowdown-src = { url = "github:kristapsdz/lowdown"; flake = false; };
   inputs.flake-compat = { url = "github:edolstra/flake-compat"; flake = false; };
+  inputs.flake-schemas.url = github:DeterminateSystems/flake-schemas;
```

```suggestion
  inputs.flake-schemas.url = "github:DeterminateSystems/flake-schemas";
```
for those of us who enable `no-url-literals`.